packages/cloud2edge/README.md

```diff
@@ -0,0 +1,44 @@
+# Eclipse Cloud2Edge
+
+The Eclipse IoT Cloud2Edge (C2E) package is an integrated suite of services developers can use to build
+IoT applications that are deployed from the cloud to the edge.
+
+The package currently consists of
+
+* Eclipse Hono
+* Eclipse Ditto
+
+The package is supposed to provide an easy way for developers to start using Eclipse Hono and Ditto in their
+IoT application.
+
+## Installing the Chart
+
+To install the chart with the release name `c2e`, run the following command (tested with Helm v3):
+
+```bash
+helm repo add eclipse-iot https://eclipse.org/packages/charts
+helm repo update
+helm install c2e eclipse-iot/cloud2edge
```

Comment:
In order for that to work we would have to host the packages also via the `https://eclipse.org/packages/charts` `index.yaml`.
I would not mix charts of single components with complete packages, that might be confusing for users (and for us maintainers).

I would host them under a separate folder, e.g. `https://eclipse.org/packages/packages` and also publish another Helm Hub alias, e.g. `eclipse-iot-packages`.
That way the "cloud2edge" could be installed via:
```
helm install c2e eclipse-iot-packages/cloud2edge
```

Reply:
> I would not mix charts of single components with complete packages, that might be confusing for users (and for us maintainers).

I have no particular preference for it. In any case, the *packages* folder needs to be included in the CI job. If we go for a separate chart repo, then we also nee to modify the web site build job on Eclipse Jenkins accordingly, right @ctron ?

Reply:
> If we go for a separate chart repo, then we also nee to modify the web site build job on Eclipse Jenkins accordingly, right @ctron ?

Yes, if we want to have an additional repository, we also need to adapt the Jenkins job. That shouldn't be a big deal though.